homeassistant/components/keba/binary_sensor.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, keba, key, sensor_name, device_class):
         """Initialize the KEBA Sensor."""
         self._key = key
         self._keba = keba
-        self._name = sensor_name
+        self._sensor_name = sensor_name
```

Comment:
Use `self._name` instead of `self._sensor_name` since it is used in every other integration

Reply:
Thanks for your comments, you are absolutely right -> changed it